src/quo2/components/tags/context_tags.cljs

```diff
@@ -64,7 +64,7 @@
        [rn/image
         {:style  {:width            20
                   :border-radius    10
-                  :background-color :red
+                  :background-color :white
```

Comment:
Hi, I changed this to `red` in https://github.com/status-im/status-mobile/pull/14626 because context tags don't have a fallback design. But not sure if the red color approach works for this component, as this is quite small. So creating an issue for that and reverting it back. (will fix https://github.com/status-im/status-mobile/issues/14706)